cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -501,18 +510,26 @@ protected void configureRoutes() {
         get(forecastFilePath, new ForecastFileController(metrics));
         addCacheControl(forecastFilePath, 1, TimeUnit.DAYS);
 
+        addWaterUserHandlers(format("/projects/{%s}/{%s}/water-user", OFFICE, PROJECT_ID), requiredRoles);
+        addWaterContractHandlers(format("/projects/{%s}/{%s}/water-user/{%s}/contracts", OFFICE, PROJECT_ID,
+                WATER_USER), requiredRoles);
+        addWaterContractTypeHandlers(format("/projects/{%s}/{%s}/water-user/{%s}/contracts/{%s}/types", OFFICE,
```

Comment:
shouldn't need contract name or water user

Reply:
Removed contract name, water user, project-id. Renamed path to "/projects/{office}/contract-types"